packages/blade/src/components/Table/_decisions/decisions.md

```diff
@@ -56,6 +56,8 @@ We don't have enough use-cases for the following features at Razorpay and hence
 - Nested Tables
 - Hiding Columns
 - Editable Rows
+- Table for React Native
+  - Mobile Apps will be using Lists instead of Tables to convey information
```

Comment:
Is this confirmed from their side? There could be potential table related UIs on mobile

Reply:
The design team did confirm. I will still tag @shridhar52war @nikhil-rzp here

Reply:
Table components are generally not used. Alternative to the table component for mobile design cards, lists, or expandable sections are used usually. It's a design call.